.pylintrc

```diff
@@ -60,7 +60,7 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 #disable=import-star-module-level,old-octal-literal,oct-method,print-statement,unpacking-in-except,parameter-unpacking,backtick,old-raise-syntax,old-ne-operator,long-suffix,dict-view-method,dict-iter-method,metaclass-assignment,next-method-called,raising-string,indexing-exception,raw_input-builtin,long-builtin,file-builtin,execfile-builtin,coerce-builtin,cmp-builtin,buffer-builtin,basestring-builtin,apply-builtin,filter-builtin-not-iterating,using-cmp-argument,useless-suppression,range-builtin-not-iterating,suppressed-message,no-absolute-import,old-division,cmp-method,reload-builtin,zip-builtin-not-iterating,intern-builtin,unichr-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,input-builtin,round-builtin,hex-method,nonzero-method,map-builtin-not-iterating
-disable=locally-disabled
+disable=locally-disabled,superfluous-parens,wrong-import-position, fixme, duplicate-code, maybe-no-member
```

Comment:
In gpkit our `pylint.sh` script runs pylint with `--disable=fixme`. That seems better than disabling here, so that pylint plugins will show todos.

Also not sure I agree with disabling `duplicate-code`, especially while we're working towards refactor! But I know it's a pain until we get the counts to zero, so maybe we should address that by opening an issue re: duplicate-code.